src/apps/backend/modules/booking/booking-service.ts

```diff
@@ -0,0 +1,21 @@
+import { CreateBookingRequestParams } from './types';
+
+export default class BookingService {
+  static async createBooking({
+    firstName,
+    phoneNumber,
+    address,
+    schedule,
+  }: CreateBookingRequestParams) {
+    if (!firstName || !phoneNumber || !address) {
+      throw new Error('Customer details are required');
+    }
+
+    // Mock booking logic, which can later be replaced with actual logic
+    const bookingDetails = {
+      confirmationMessage: `Your booking has been confirmed for the time slot from ${schedule.startTime} to ${schedule.endTime}`,
+    };
+
+    return Promise.resolve(bookingDetails);
```

Comment:
`BookingService` should return an instance of `Booking`.

The controller in vapi module should be responsible to serialize the response in a way that Vapi needs.